build/build.proj

```diff
@@ -355,19 +355,19 @@
 
     <PropertyGroup>
       <!-- Sort assemblies -->
-      <DesktopInputTestAssemblies>@(TestAssemblyPath->WithMetadataValue("IsDesktop", "true"))</DesktopInputTestAssemblies>
-      <DesktopInputTestAssembliesSpaced>$(DesktopInputTestAssemblies.Replace(';', ' '))</DesktopInputTestAssembliesSpaced>
-      <CoreInputTestAssemblies>@(TestAssemblyPath->WithMetadataValue("IsCore", "true"))</CoreInputTestAssemblies>
-      <CoreInputTestAssembliesSpaced>$(CoreInputTestAssemblies.Replace(';', ' '))</CoreInputTestAssembliesSpaced>
+      <DesktopInputTestAssemblies Condition=" '$(RunTestType)' == 'IsDesktop' ">@(TestAssemblyPath->WithMetadataValue("IsDesktop", "true"))</DesktopInputTestAssemblies>
```

Comment:
Actually, can we have 2 different property names for running desktop and running core?

RunCore and RunDesktop or some better name that makes sense in that context. 

That way, if I want to I can trigger both core and desktop myself. 

Similarly, running it locally would just trigger both, rather than nothing like it would right now.

Reply:
Refactored to allow IsCore and IsDesktop flags to be set independently.